compiler/riscv/codegen/GenerateInstructions.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019, 2022 IBM Corp. and others
+ * Copyright (c) 2019, 2020 IBM Corp. and others
```

Comment:
This should be 2022, as it was originally.

Reply:
Oops, I overlooked this. The commit is actually much older and I just rebased...fixed in new version.